### PR TITLE
[oneDNN] Code cleanup: unsupport blocked format for fused batchnorm op

### PR DESCRIPTION
**Pull request by gzmkl**
We will not support blocked format for MKL kernel operations.

This PR does related code cleanup of the FusedBatchNorm op and related tests.

### CODE REVIEW

**Comment by gzmkl** [comment]
@penpornk Hi, please help to review this and another PRs. Once they are approved/merged, I have another one (big cleanup) for TF 2.17. Thanks a lot!

**Comment by gbaned** [comment]
Hi @gzmkl Can you please assist on above [comments](https://github.com/tensorflow/tensorflow/pull/66388#issuecomment-2103579140) from @gzmkl.  Thank you!

**Comment by gzmkl** [comment]
> Hi @gzmkl Can you please assist on above [comments](https://github.com/tensorflow/tensorflow/pull/66388#issuecomment-2103579140) from @gzmkl. Thank you!

Hi, that are comments from myself! Right now, we are planning to remove a lot of very old code which is N/A forever.

**Comment by gbaned** [comment]
> > Hi @gzmkl Can you please assist on above [comments](https://github.com/tensorflow/tensorflow/pull/66388#issuecomment-2103579140) from @gzmkl. Thank you!
> 
> Hi, that are comments from myself! Right now, we are planning to remove a lot of very old code which is N/A forever.

Hi @gzmkl My apologies for the typo in my above comment. Thank you very for much the update.

**Comment by gbaned** [comment]
Hi @penpornk Any update on this PR? Please. Thank you!

**Comment by keerthanakadiri** [comment]
Hi @penpornk Any update on this PR? Please. Thank you!

**Comment by keerthanakadiri** [comment]
Hi @penpornk Any update on this PR? Please. Thank you!

**Comment by keerthanakadiri** [comment]
Hi @penpornk, Any update on this PR? Please. Thank you!

**Comment by keerthanakadiri** [comment]
Hi @penpornk, Any update on this PR? Please. Thank you!

**Comment by keerthanakadiri** [comment]
Hi @penpornk, Any update on this PR? Please. Thank you!

**Comment by gzmkl** [comment]
Closed this PR because there is no review activity for too long!
